### DISCUSSION TITLE
Add Community Program Repository for Searching and Sharing Programs

### DISCUSSION THREAD

**Discussion by feraask** [astashov/liftosaur#148] [Ideas]
Would be really cool if there was a central repository where people can share their Liftosaur programs. Rather than just generating a link and posting it in the #routines Discord or on Reddit.

Something like an option in the app to mark a program as private or public.

Add some configurable properties like goals (hypertrophy, strength, etc...), difficulty/experience level, male/female/unisex focused, required equipment, description/documentation using markdown, and whatever else to describe the program.

Then having a dedicated section in the app or website where you can search and filter the public programs from the community and easily preview them and import them into your account. Similar to how [Boostcamp does it](https://www.boostcamp.app/programs).

Very long-term, having the option to make free/paid programs where you have the option to keep the Liftoscript code private/hidden but the program creator can decide to allow some basic modifications like reordering/swapping exercises (perhaps only for the same muscle group or something). Basically turning it into a full program marketplace.

**Comment by Bruno-366** [comment]
For now this Reddit thread seems to be the main place that people are sharing their programs:

[🏋️ Share your Liftosaur programs here! : r/liftosaur](https://www.reddit.com/r/liftosaur/comments/18vicce/share_your_liftosaur_programs_here/)

Linking to it here for anyone else that was, like me, wondering where to find community created programs 😄
